shell/platform/windows/window_binding_handler_delegate.h

```diff
@@ -22,6 +22,10 @@ class WindowBindingHandlerDelegate {
   // called on the platform thread.
   virtual void OnWindowSizeChanged(size_t width, size_t height) = 0;
 
+  // Notifies delegate that backing window needs to be repainted.
+  // Typically called by currently configured WindowBindingHandler
```

Comment:
Nit: Missing period. (This is true of all the other places it was copy/pasted to in the past; bonus points for fixing those as well 🙂 )

Reply:
Done.
Add missing period for all those lines in the file.